desktop/mac/release-notes/index.md

```diff
@@ -33,6 +33,33 @@ Take a look at the [Docker Public Roadmap](https://github.com/docker/roadmap/pro
 > chip](https://desktop.docker.com/mac/main/arm64/Docker.dmg?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-mac-arm64){:
 > .button .primary-btn }
 
+### New
+
```

Comment:
Suggest a new entry here for New Docker menu

- Docker Desktop 4.4.5 introduces a new version of the Docker menu which creates a consistent user experience across all operating systems.

Reply:
Added